Content : HTML CSS Week 1 (#188)
CodeYourFuture/curriculum: content/blocks/spaced-repetition/index.md

+++
title="Spaced Repetition"
headless="true"
time= 10
+++

{{<tabs name="Scheduled check-ins">}}
{{<tab name="Text">}}

> ## What are we doing now?

Author: It will once the new tabs are merged. I wrote it assuming the new tabs would go through.

> ## What are we doing now?

Author: fixed by #198

You're going to use this prep time to answer some questions _and schedule in times to review your understanding_. In your planner or calendar, schedule a short (10 minute) review of your answers:

{{<note type="tip" title="Space at increasing intervals">}}

- One week from today
- One month from today
- Three months from today
- Six months from today
- One year from today

{{</note>}}

Do this now. Then move forward to working through as many of these questions as you can. (You may not complete them all, and that is ok.)

> _Write down your answers first_ and then update them after looking in the details, if it's useful to do so. Use all available tools to help you answer, and remember:

Author: ```suggestion
_Write down your answers first_ and then update them after looking at the sample answers, if it's useful to do so. Use all available tools to help you answer, and remember:
```

> the point _is not_ to complete the exercises
> the point is for you to secure your own understanding

### Examples of tools you might use

- [MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS)

> - [A large language model like ChatGPT](https://chat.openai.com/)

Author: I disagree. (I mean, I don't disagree it's full of plausible lies! It's got huge problems!)

But we have to encourage it because they are all using it and the world of work many of them will be entering in a year or so will expect them to know how to use it well. 

So we need to talk about how to use it well and place it in context with fact-checking, testing, and judgment. And we will need some workshops on this! (I have some upcoming but welcome many more.)

> - [A large language model like ChatGPT](https://chat.openai.com/)

Reviewer: This is a wider discussion we should generally have, and have a strategy for - I'll mark as resolved because this PR shouldn't be blocked by it, but let's keep talking as a group :)

> - [A large language model like ChatGPT](https://chat.openai.com/)

Reviewer: Also, having now read the workshop, I think the way it treats the topic is pretty good :)

- [Devtools](https://developer.chrome.com/docs/devtools/)
  {{</tab>}}

{{<tab name="Video">}}
{{<youtube>}}

> https://youtu.be/U1QWx8xbbt4

Author: It will...
